### ISSUE TITLE
hard to train to see the pole object in the CamVid

### ISSUE THREAD

**Issue by guanfuchen** [fyu/drn#15]
![image](https://user-images.githubusercontent.com/22321977/34460000-e3581bae-ee3b-11e7-9cc0-b9ae0eb8089d.png)

Thanks for your code using drn to segment the CamVid dataset, but I use the code well on most objects segmenting, it is hard to segment the pole object in the CamVid dataset, so what can I do to get the pole object to get segmenting.

**Comment by fyu** [comment]
What's your training setting? Image resolution?

**Comment by guanfuchen** [comment]
I use learning rate=1e-5, momentum=0.99, weight_decay=5e-4, and use SGD solver, batch_size=1, Image resolution is 360*480.
 


-----原始邮件-----
发件人:"Fisher Yu" <notifications@github.com>
发送时间:2017-12-31 15:18:51 (星期日)
收件人: fyu/drn <drn@noreply.github.com>
抄送: guanfuchen <guanfuchen@zju.edu.cn>, Author <author@noreply.github.com>
主题: Re: [fyu/drn] hard to train to see the pole object in the CamVid (#15)



What's your training setting? Image resolution?
